src/engraving/layout/v0/chordlayout.cpp

```diff
@@ -2161,6 +2161,98 @@ static std::pair<double, double> layoutAccidental(const MStyle& style, AcEl* me,
     return std::pair<double, double>(me->x, me->x + me->width);
 }
 
+//---------------------------------------------------------
+//   placeDots
+//---------------------------------------------------------
+
+void ChordLayout::placeDots(std::vector<Chord*> chords, std::vector<Note*> notes)
```

Comment:
**const** std::vector<Chord*>**&** chords, **const** std::vector<Note*>**&** notes

Reply:
You missed &

Reply:
@asattely please fix this ^ and open the same PR for 4.1.0

Reply:
done, and https://github.com/musescore/MuseScore/pull/18419